lib/index.js

```diff
@@ -120,7 +120,9 @@ const downloadFromInfoCallback = (stream, info, options) => {
       backoff: { inc: 500, max: 10000 },
     });
 
-    const shouldBeChunked = !format.hasAudio || !format.hasVideo;
+    let shouldBeChunked;
+    if (options.dlChunkSize === 0) shouldBeChunked = false;
+    else shouldBeChunked = !format.hasAudio || !format.hasVideo;
```

Comment:
can be `let shouldBeChunked = dlChunkSize !== 0 && (!format.hasAudio || !format.hasVideo)`

or `dlChunkSize > 0`

Reply:
Done